src/custom-element.js

```diff
@@ -1661,6 +1661,8 @@ function createBaseCustomElementClass(win) {
       return (
         // in FIE
         (this.ampdoc_ && this.ampdoc_.win != this.ownerDocument.defaultView) ||
+        // TODO(#22733): cleanup once the ampdoc-fie is fully launched.
+        (this.ampdoc_ && this.ampdoc_.getParent()) ||
```

Comment:
Since the return type is strictly boolean, we need to do `!!this.ampdoc_.getParent()`. Otherwise it'd return the actual parent object as a truthy value.

Reply:
done.